sdk/eventgrid/eventgrid/test/public/eventGridClient.spec.ts

```diff
@@ -239,7 +239,7 @@ describe("EventGridPublisherClient", function (this: Suite) {
       assert.equal(spans.length, 3);
       assert.equal(spans[0].name, "root");
       assert.equal(spans[1].name, "Azure.Data.EventGrid.EventGridPublisherClient-send");
-      assert.equal(spans[2].name, "/api/events");
+      assert.equal(spans[2].name, "HTTP POST");
```

Comment:
@ellismg another approach I was thinking of taking here is removing this assertion as we are now testing the core tracingPolicy in a client package. On the other hand, it also asserts that no intermediary spans are created so maybe it's a good assertion to have.

So, what do you think? I vote deleting this assertion, but will defer to you

Reply:
Deleting it is fine with me - your justification makes sense!